### PR TITLE
Azure sql statements

### PR DESCRIPTION
**Pull request by TylerMetz**
## Describe your changes
Redid our schema in the backend Azure database

## Issue ticket number and link
#438

### CODE REVIEW

**Review by rileycleavenger — APPROVED** [review]
LGTM!!!!!!!!!!

**Review by samforstot — APPROVED** [review]
LGTM! Looks way cleaner compared to what we had before. Can't wait to see it implemented

**Comment by TylerMetz** [comment]
@samforstot azure on top!

**Review by steeleelliott03 — APPROVED** [review]
dazzler
LGTM

File: Other/BackendDocumentation/SQL/SQL Statements/TableCreation.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# MealDealz Schema V2
+
+## Deals Data
+
+```SQL
+IF NOT EXISTS (SELECT * FROM dbo.tables WHERE name = 'deals_data'),
+                BEGIN
+                    CREATE TABLE deals_data (
+                        store VARCHAR(25) PRIMARY KEY,
+                        foodName VARCHAR(50),
```
Comment (rileycleavenger) [inline Other/BackendDocumentation/SQL/SQL Statements/TableCreation.md:10]: this one might need to be a bit longer

File: Other/BackendDocumentation/SQL/SQL Statements/TableCreation.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+```
+
+## User Cookies
+
+```SQL
+IF NOT EXISTS (SELECT * FROM dbo.tables WHERE name = 'user_cookies')
+                BEGIN
+                    CREATE TABLE user_cookies (
+                        UserName VARCHAR(25) PRIMARY KEY,
+                        Cookie VARCHAR(255),
```
Comment (rileycleavenger) [inline Other/BackendDocumentation/SQL/SQL Statements/TableCreation.md:65]: why is this 255

File: Other/BackendDocumentation/SQL/SQL Statements/TableCreation.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+## User Recipes
+
+```SQL
+IF NOT EXISTS (SELECT * FROM dbo.tables WHERE name = 'user_recipes')
+                BEGIN
+                    CREATE TABLE user_recipes (
+                        RecipeID INT PRIMARY KEY IDENTITY(1,1),
+                        Title VARCHAR(25),
+                        Ingredients VARCHAR(255),
```
Comment (rileycleavenger) [inline Other/BackendDocumentation/SQL/SQL Statements/TableCreation.md:93]: I found one set of ingredients in the json file that is 970 characters just from scrolling, and where the instructions are 6819 characters

Comment (rileycleavenger) [reply Other/BackendDocumentation/SQL/SQL Statements/TableCreation.md:93]: I thinks ingredients should be 1500 and instructions probably 10000

File: Other/BackendDocumentation/SQL/SQL Statements/TableCreation.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+```
+
+## Jason's Recipes
+```SQL
+IF NOT EXISTS (SELECT * FROM dbo.tables WHERE name = 'jason_recipes')
+                BEGIN
+                    CREATE TABLE jason_recipes (
+                        RecipeID INT PRIMARY KEY IDENTITY(1,1),
+                        Title VARCHAR(25),
+                        Ingredients VARCHAR(255),
```
Comment (rileycleavenger) [inline Other/BackendDocumentation/SQL/SQL Statements/TableCreation.md:107]: same thing here, 1500 for ingredients and prob 10k for instructions

**Review by rileycleavenger — COMMENTED** [review]
adjust these in the tables

**Pull request merged**